scripts/build-container-image.sh

```diff
@@ -81,24 +94,30 @@ cleanup_old_tags() {
   fi
 }
 
-get_image_latest_tag() {
-  echo "$(git_get_current_branch)_latest"
-}
-
 get_image_path() {
   image_tag=${1:?}
   echo "${HUB}/${HUB_ORG}/${IMAGE_NAME}:${image_tag}"
 }
 
 build_container_image() {
-  image_tag=$(get_image_tag)
-  image_latest_tag=$(get_image_latest_tag)
-  image_path=$(get_image_path "${image_tag}")
-  image_latest_path=$(get_image_path "${image_latest_tag}")
+  image_path=$(get_image_path "${GIT_REFERENCE}")
```

Comment:
```suggestion
  declare -a image_tags=()

  image_path=$(get_image_path "${GIT_REFERENCE}")
```
You want to declare your arrays.